src/server/v2.0/handler/scan_all.go

```diff
@@ -54,6 +56,31 @@ func (s *scanAllAPI) Prepare(ctx context.Context, operation string, params inter
 	return nil
 }
 
+// StopScanAll stops the execution of scan all artifacts.
+func (s *scanAllAPI) StopScanAll(ctx context.Context, params operation.StopScanAllParams) middleware.Responder {
+	if err := s.requireAccess(ctx, rbac.ActionStop); err != nil {
+		return s.SendError(ctx, err)
+	}
+
+	execution, err := s.getLatestScanAllExecution(ctx)
+	if err != nil {
+		return s.SendError(ctx, err)
+	}
+	if execution == nil {
+		message := fmt.Sprintf("no scan all job is found currently")
+		return s.SendError(ctx, errors.BadRequestError(nil).WithMessage(message))
+	}
+	go func(ctx context.Context, eid int64) {
+		err = s.execMgr.Stop(ctx, eid)
```

Comment:
`err = s.execMgr.Stop(ctx, eid)` => `err := s.execMgr.Stop(ctx, eid)` please don't overwrite the err out of the scope.

Reply:
nice catch. thanks